website/client/components/chat/chatCard.vue

```diff
@@ -273,5 +273,8 @@ export default {
       return habiticaMarkdown.render(String(text));
     },
   },
+  mounted () {
+    this.$emit('item-mounted', this.msg.id);
```

Comment:
$emit is broadcasted to all components upstream? In that case a more detailed name might help to avoid confusion like `message-component-mounted`

Reply:
not to all components, just to the ones that use this event inside the template